examples/1bitsy/tft-st7735/systick.h

```diff
@@ -0,0 +1,25 @@
+#ifndef SYSTICK_included
```

Comment:
License & Copyright header is missing.

Reply:
I copied this from the  tft1 example and it didn't have a copyright notice. I'll add a libopencm3 piotr notice.